homework/english/kataya_pechinskaya/lecture1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+#Homework 2 for lesson 1 ( voabulary)
+1.	Every time when a new employee comes in the company, administration staff BOOT allows Windows 10 pro and INSTALL the computer.
+2.	The management gave the order CREATE a new virtual machine. 
+3.	I need TO CONNECT to this English lecture in Zoom.
+4.	My program has bug, I mast do DEBUG my program, if I would like it run.
+5.	Antivirus is searching virus in computer and DELETING it.
+6.	In finish course all students will DEVELOP their chatbot.
+7.	Site is DISCONNECT, system administrator needs DISABLE and ENABLE internet connection for access site.
+8.	In meeting I DISPLAYED my week work.
+9.	My program can EXECUTE mathematic operation. 
+10.	At my work we IMPLEMENTED system UPDATES for all computers in company.
+11.	Yesterday I bought a new laptop and INSTALL Windows 10 and Microsoft Office.
+12.	 My friends recommended me PLUG IN new Internet package for quicker speed that now.
+13.	My laptop catches in virus I need TO RESTORE operating system.
+14.	I don’t use a program Opera and UNINSTALL it.
```

Comment:
maybe you want to say 'I don’t use a program Opera so UNINSTALL it'
sorry, if i miss